phpunit

```diff
@@ -52,6 +52,8 @@ if (!file_exists("$PHPUNIT_DIR/phpunit-$PHPUNIT_VERSION/phpunit") || md5_file(__
     $zip->close();
     chdir("phpunit-$PHPUNIT_VERSION");
     passthru("$COMPOSER remove --no-update symfony/yaml");
+    passthru("$COMPOSER remove --no-update phpspec/prophecy");
+    passthru("$COMPOSER require --no-update phpunit/phpunit-mock-objects \"3.1.1\"");
```

Comment:
This change should not be done for PHPUnit 4.8, as it depends on ``phpunit/phpunit-mock-objects ~2.3``